config/kubernetes/sig-docs/teams.yaml

```
@@ -1,4 +1,18 @@
 teams:
+  sig-docs-bn-owners:
+    description: Admins for Bengali content
```

In line with the rest of the document, this should read as

```suggestion
    description: Approvers for Bengali content
```

---

Okay sure!